commands/General/ping.js

```diff
@@ -3,7 +3,7 @@ const { MessageEmbed } = require("discord.js")
 
 module.exports.run = async (client, message, args) => {
     message.channel.send({
-        embeds: [new MessageEmbed().setTitle("Pong!").setColor("RED").setDescription(`Ping: ${client.ws.ping}ms`)]
+        embeds: [new MessageEmbed().setTitle("Pong!").setColor("#36393F").addField(`> <:pepehappy:759028157898293300> API Latency:`, `**${client.ws.ping}ms**`, true).addField(`> <:blobthumbsuo:759028035822813214> Bot Latency:`, `**${Date.now() - message.createdTimestamp}ms**`, true)]
```

Comment:
Does the bot has access to the emojis?

Reply:
It should

Reply:
The test bot isn't in the normal DBH's server

Reply:
why do we need them anyways

Reply:
If there are emojis they should be normal discord ones not custom, this way people don't got to change hardcoded emoji Id's

Reply:
i don't think that bot needs to have a bunch of emojis in every command